mesheryctl/internal/cli/root/system/start.go

```diff
@@ -86,7 +86,7 @@ func start() error {
 		}
 	}
 
-	if _, err := os.Stat(utils.MesheryFolder); os.IsNotExist(err) {
+	if _, err := os.Stat(utils.DockerComposeFile); os.IsNotExist(err) {
```

Comment:
What happens if the MesheryFolder itself doesn't exist?

Reply:
the above check is for that

Reply:
look into the condition above

Reply:
Thanks, @anirudhjain75. I missed the check for the presence of the directory above.

@navendu-pottekkat, a mental note to take into consideration for when all of the K8s manifests are retrieved.

Reply:
@leecalcote Noted.